code/game/objects/items/melee/batons/classic.dm

```diff
@@ -0,0 +1,133 @@
+/obj/item/melee/classic_baton
+	name = "police baton"
+	desc = "A wooden truncheon for beating criminal scum."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "baton"
+	item_state = "classic_baton"
+	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
+	slot_flags = ITEM_SLOT_BELT
+	force = 12 //9 hit crit
+	w_class = WEIGHT_CLASS_NORMAL
+	var/cooldown = 13
+	var/last_hit = 0
+	var/stun_stam_cost_coeff = 1.25
+	var/hardstun_ds = 1
+	var/softstun_ds = 0
+	var/stam_dmg = 30
+
+/obj/item/melee/classic_baton/attack(mob/living/target, mob/living/user)
+	if(!baton_stun(target, user))
+		return ..()
+
+// return true to disrupt normal melee attack chain
+/obj/item/melee/classic_baton/proc/baton_stun(mob/living/target, mob/living/user)
+	if(user.getStaminaLoss() >= STAMINA_SOFTCRIT)//CIT CHANGE - makes batons unusuable in stamina softcrit
+		to_chat(user, "<span class='warning'>You're too exhausted for that.</span>")//CIT CHANGE - ditto
+		return FALSE//CIT CHANGE - ditto
+
+	add_fingerprint(user)
+	if((HAS_TRAIT(user, TRAIT_CLUMSY)) && prob(50))
+		to_chat(user, "<span class ='danger'>You club yourself over the head.</span>")
+		user.Knockdown(60 * force)
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			H.apply_damage(2*force, BRUTE, BODY_ZONE_HEAD)
+		else
+			user.take_bodypart_damage(2*force)
+		return TRUE
+	if(iscyborg(target))
+		return FALSE
+	if(!isliving(target))
+		return FALSE
+	if (user.a_intent == INTENT_HARM)
+		return FALSE
+	else
```

Comment:
Remove this `else` that only makes things more indentated, the previous if statements lead to early returns anyway.